src/traces/table/plot.js

```diff
@@ -0,0 +1,82 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var table = require('./render');
+
+module.exports = function plot(gd, cdTable) {
+
+    var fullLayout = gd._fullLayout;
+    var svg = fullLayout._paper;
+    var root = fullLayout._paperdiv;
+
+    var gdColumns = {};
+    var gdColumnsOriginalOrder = {};
+
+    var size = fullLayout._size;
+
+    cdTable.forEach(function(d, i) {
+        gdColumns[i] = gd.data[i].header.values.map(function(d) {return d[0];});
+        gdColumnsOriginalOrder[i] = gdColumns[i].slice();
+    });
+
+    var hover = function(eventData) {
+        gd.emit('plotly_hover', eventData);
```

Comment:
Before merging, we should make sure `eventData` is on par with cartesian traces.

See https://github.com/plotly/plotly.js/pull/1896 and https://github.com/plotly/plotly.js/pull/1770